### PR TITLE
Update oFono from SFOS

### PR DESCRIPTION
**Pull request by Herrie82**
@Tofee still WIP, untested, but should get us in a better state.

### CODE REVIEW

File: meta-luneos/recipes-connectivity/ofono/ofono_%.bbappend [hunk 1]
```diff
@@ -9,9 +9,13 @@ PV:halium = "1.29+git${SRCPV}"
 LICENSE = "GPL-2.0-only"
 LIC_FILES_CHKSUM = "file://COPYING;md5=eb723b61539feef013de476e68b5c50a"
 
-DEPENDS += "libglibutil dbus-glib libgrilio libmce-glib"
+DEPENDS += "dbus-glib libmce-glib"
 RDEPENDS:${PN} += "mobile-broadband-provider-info ofono-conf"
 
+#For Halium 9 devices we want to use the ofono-binder-plugin, for older devices we might want to use ofono-ril-plugin
+RDEPENDS:${PN}:append:halium = " ofono-binder-plugin"
```
Comment (Herrie82) [inline meta-luneos/recipes-connectivity/ofono/ofono_%.bbappend:16]: @Tofee this should fix it :) I forgot an :append: there in between, so both above RDEPENDS would not get installed.

**Comment by Herrie82** [comment]
@Tofee I think with latest pushes now this should be OK to be merged.

File: meta-luneos/recipes-connectivity/ofono/ofono-binder-plugin.bb [hunk 2]
```diff
@@ -1,11 +1,11 @@
 # Copyright (c) 2023 Herman van Hazendonk <github.com@herrie.org>
 
-DESCRIPTION = "Binder based ofono plugin (Needed for Android 9.0+)."
-LICENSE = "GNUPL"
+DESCRIPTION = "Binder based ofono plugin (Needed for Android 8.0+)."
+LICENSE = "GPL-2.0-only"
+LIC_FILES_CHKSUM = "file://${COMMON_LICENSE_DIR}/GPL-2.0-only;md5=801f80980d171dd6425610833a22dbe6"
```
Comment (shr-project) [inline meta-luneos/recipes-connectivity/ofono/ofono-binder-plugin.bb:5]: Don't use COMMON_LICENSE_DIR

Comment (Herrie82) [reply meta-luneos/recipes-connectivity/ofono/ofono-binder-plugin.bb]: @shr-project fixed!

File: meta-luneos/recipes-connectivity/ofono/libofonobinderpluginext.bb [hunk 3]
```diff
@@ -0,0 +1,27 @@
+# Copyright (c) 2023 Herman van Hazendonk <github.com@herrie.org>
+
+DESCRIPTION = "Extension framework for ofono binder plugin"
+LICENSE = "GPL-2.0-only"
+LIC_FILES_CHKSUM = "file://${COMMON_LICENSE_DIR}/GPL-2.0-only;md5=801f80980d171dd6425610833a22dbe6"
```
[the remark this answers is not in the document]
Comment (Herrie82) [reply meta-luneos/recipes-connectivity/ofono/libofonobinderpluginext.bb]: Fixed

**Comment by Tofee** [comment]
LGTM. @shr-project all good for you too?

**Review by shr-project — APPROVED** [review]

**Pull request merged**